### PR TITLE
fix_integrationtests_for_cn

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

**Review by spyrostz — APPROVED** [review]
LGTM

**Review by faizan2590 — APPROVED** [review]
lgtm

File: src/d3a/models/strategy/load_hours.py [hunk 1]
```diff
@@ -261,6 +262,7 @@ def _one_sided_market_event_tick(self, market, offer=None):
                     return
                 acceptable_offer = offer
             current_day = self._get_day_of_timestamp(market.time_slot)
+            print(market.time_slot_str, current_day)
```
Comment (faizan2590) [inline src/d3a/models/strategy/load_hours.py:265]: please remove, before merging!

**Review by faizan2590 — DISMISSED** [review]
LGTM

**Pull request merged**